stacks/SiteStack.ts

```diff
@@ -79,6 +79,7 @@ export function SiteStack({ stack }: StackContext) {
                     "dynamodb:Query",
                     "dynamodb:BatchGetItem",
                     "dynamodb:BatchWriteItem",
+                    "dynamodb:Scan",
```

Comment:
what is this permission for?

Reply:
Currently there is an issue in Test for obtaining org data in page `manage-organisation`. This page pull in the required data using scan operation. 

This was to fix an existing issue in test env and not related to the Jira task meant for this PR